### PR TITLE
base PR for set polarion as automated

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

File: apps/polarion/polarion_utils.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 import shlex
 import subprocess
-from pylero.exceptions import PyleroLibException
 
+from apps.utils import get_util_config
 
 LOGGER = get_logger(name=__name__)
+AUTOMATED = "automated"
+NOT_AUTOMATED = "notautomated"
+APPROVED = "approved"
```
Comment (myakove) [inline apps/polarion/polarion_utils.py:13]: Duplicate code.

File: apps/polarion/polarion_utils.py [hunk 2]
```diff
@@ -19,31 +25,74 @@ def git_diff_lines():
...
-
+        if line.startswith("-"):
+            diff.setdefault("removed", []).append(line)
     return diff
 
 
 def validate_polarion_requirements(
-    polarion_test_ids,
     polarion_project_id,
+    polarion_test_ids=None,
```
Comment (myakove) [inline apps/polarion/polarion_utils.py:35]:
```suggestion
    polarion_test_ids=None,
```
Why this is None?
The function do nothing if this is None

File: pyproject.toml [hunk 3]
```diff
@@ -38,7 +38,7 @@ packages = [{ include = "apps" }]
 [tool.poetry.scripts]
 pyutils-unusedcode = "apps.unused_code.unused_code:get_unused_functions"
 pyutils-polarion-verify-tc-requirements = "apps.polarion.polarion_verify_tc_requirements:has_verify"
-
+pyutils-polarion_set_automated = "apps.polarion.polarion_set_automated:polarion_approve_automate"
```
Comment (myakove) [inline pyproject.toml:41]: should be `-`

File: tests/polarion/test_verify_polarion_requirements.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import subprocess
 from pyhelper_utils.shell import run_command
 
-BASE_COMMAND = "poetry run python apps/polarion/polarion_verify_tc_requirements.py"
+BASE_COMMAND = "poetry run python3 apps/polarion/polarion_verify_tc_requirements.py"
```
Comment (myakove) [inline tests/polarion/test_verify_polarion_requirements.py:5]: why `python3`?

File: tests/polarion/test_polarion_automated.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    command = f"{BASE_COMMAND} --config-file-path=config.example.yaml"
+    rc, _, err = run_command(
+        command=shlex.split(command),
+        verify_stderr=False,
+        check=False,
+        capture_output=False,
+        stderr=subprocess.PIPE,
+    )
+    assert not err
+    assert rc
```
Comment (myakove) [inline tests/polarion/test_polarion_automated.py:30]: already tested.

**Review by myakove — CHANGES_REQUESTED** [review]

File: tests/polarion/test_verify_polarion_requirements.py (hunk 4 above)
Comment (dbasunag) [reply tests/polarion/test_verify_polarion_requirements.py:5]: This is not needed, I will undo.

File: tests/polarion/test_polarion_automated.py (hunk 5 above)
Comment (dbasunag) [reply tests/polarion/test_polarion_automated.py:30]: base command is different though.

File: apps/polarion/polarion_utils.py (hunk 1 above)
Comment (dbasunag) [reply apps/polarion/polarion_utils.py:13]: Removed.

File: tests/polarion/test_polarion_automated.py (hunk 5 above)
Comment (myakove) [reply tests/polarion/test_polarion_automated.py:30]: this do not test anything (beside click), the uderline commands already tested.

Comment (dbasunag) [reply tests/polarion/test_polarion_automated.py:30]: Ok, then I can remove it.

**Pull request merged**